Tests/KlaviyoCoreTests/TestUtils.swift

```diff
@@ -95,6 +95,7 @@ extension KlaviyoEnvironment {
             emitDeveloperWarning: { _ in },
             networkSession: { NetworkSession.test() },
             apiURL: { URLComponents(string: "https://dead_beef")! },
+            cdnURL: { URLComponents(string: "https://dead_beef")! },
```

Comment:
nit: I'd avoid using the same value for test stubs, else you could mix these properties up and a test won't catch it.